users/config.rst

```diff
@@ -612,37 +698,61 @@ address
             <address>dynamic</address>
         </device>
 
-paused
+.. option:: device.paused
+
     True if synchronization with this devices is (temporarily) suspended.
 
-allowedNetwork
-    If given, this restricts connections to this device to only this network
-    (see :ref:`allowed-networks`).
+.. option:: device.allowedNetwork
+
+    If given, this restricts connections to this device to only this network.
+    The mechanism is described in detail in a :doc:`separate chapter
+    </advanced/device-allowednetworks>`).
+
+.. option:: device.autoAcceptFolders
+
+    If ``true``, folders shared from this remote device are automatically added
+    and synced locally under the :opt:`default path <defaults.folder>`.  For the
+    folder name, Syncthing tries to use the label from the remote device, and if
+    the same label already exists, it then tries to use the folder's ID.  If
+    that exists as well, it just logs an error.  A local folder already added
```

Comment:
I think it actually asks to add the folder manually in such a case? Does it still log an error? It's been a while since I tested these, but I don't remember seeing any actual errors. I may be wrong, of course, but if possible, please re-check this one.

Reply:
You're right.  I just checked the code again and as far as I can tell it just gets the normal treatment.  The log message is just at INFO level.